api/proto/teleport/dynamicwindows/v1/dynamicwindows_service.proto

```diff
@@ -35,6 +35,8 @@ service DynamicWindowsService {
   rpc CreateDynamicWindowsDesktop(CreateDynamicWindowsDesktopRequest) returns (types.DynamicWindowsDesktopV1);
   // UpdateDynamicWindowsDesktop updates an existing dynamic Windows desktop.
   rpc UpdateDynamicWindowsDesktop(UpdateDynamicWindowsDesktopRequest) returns (types.DynamicWindowsDesktopV1);
+  // UpdateDynamicWindowsDesktop updates an existing dynamic Windows desktop or creates new if it doesn't exist.
```

Comment:
```suggestion
  // UpsertDynamicWindowsDesktop updates an existing dynamic Windows desktop or creates new if it doesn't exist.
```

But why do we need upsert?

Reply:
As mentioned by @rosstimothy in https://github.com/gravitational/teleport/pull/46988 `Update` will fail if revision doesn't match, so `tctl --force` might not be as forcing as expected